src/torcontrol.cpp

```diff
@@ -595,13 +603,7 @@ void TorController::protocolinfo_cb(TorControlConnection& conn, const TorControl
                 }
             }
         } else if (methods.count("HASHEDPASSWORD")) {
-            if (!torpassword.empty()) {
-                LogPrint("tor", "tor: Using HASHEDPASSWORD authentication\n");
-                boost::replace_all(torpassword, "\"", "\\\"");
-                conn.Command("AUTHENTICATE \"" + torpassword + "\"", boost::bind(&TorController::auth_cb, this, _1, _2));
-            } else {
-                LogPrintf("tor: Password authentication required, but no password provided with -torpassword\n");
-            }
+            LogPrintf("tor: Only hashed password authentication supported, but no password provided with -torpassword\n");
```

Comment:
This seems a bit misleading: couldn't there be supported authentication mechanism left which we don't support? Suggestion: "Only supported remaining authenthentication mechanism left is password, but ..."